.gitlab-ci.yml

```diff
@@ -1786,7 +1786,7 @@ deploy_windows_testing-a7:
   before_script:
     - rsync -azr --delete ./ $SRC_PATH
     - export TEST_PLATFORMS="sles-12,urn,SUSE:SLES-BYOS:12-SP4:2019.11.13"
-    - export TEST_PLATFORMS="$TEST_PLATFORMS|sles-15,urn,SUSE:SLES-BYOS:15:2019.11.15"
+    - export TEST_PLATFORMS="$TEST_PLATFORMS|sles-15,urn,SUSE:sles-15-sp2-byos:gen1:2020.09.21"
```

Comment:
SP2 should supersede SP1, but there will always be hosts that get kept on previous SPs. That said, we don't test all SPs / minor versions for all other OSes, so I don't think we need to here either (+ it would increase the burden on the testing infra).

Reply:
Thanks for adding the milestone and labels, Julien.

I'm kicking off another build now.